src/shared-components/Navigation/TopNavigation/Transaction_filterTypeNaigation.tsx

```diff
@@ -0,0 +1,51 @@
+import { Badge, Stack, styled, Typography } from '@mui/material';
+import FilterListIcon from '@mui/icons-material/FilterList';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { theme } from 'src/styles/theme';
+import { colors } from 'src/styles/colors';
+
+interface Transaction_filterTypeNavigationProps {
+  month: string;
+  badgeContent: number;
+}
+
+const CustomBadge = styled(Badge)(({ theme }) => ({
+  '& .MuiBadge-badge': {
+    backgroundColor: colors.violet[100],
+    color: colors.light[100],
+    borderRadius: '50%',
+    right: theme.spacing(3),
+    top: theme.spacing(3),
+    padding: theme.spacing(2),
+  },
+}));
+
+export const Transaction_filterTypeNavigation = ({
+  month,
```

Comment:
Let's also rename this prop to 'interval'

Reply:
Fixed